wp-content/plugins/core/src/Blocks/Types/Post_List/Post_List.php

```diff
@@ -0,0 +1,213 @@
+<?php
```

Comment:
Nice, I'll likely just turn this into the "Content" block when we get there. I'm wondering if we should turn this "field" into something more modular perhaps where it could be easier to deploy as needed for any given block?

Reply:
I'm not 100% sure I'm following. Do you mean breaking out the Post Query functionality so it can be re-used in other blocks?

Reply:
Sorry didn't explain it well. Yes, basically turning this overall post list field itself, which currently consists of multiple fields, either into one field or one partial to more easily be deployed across blocks as you know it will be.

Reply:
Ah yes, 100%. I'd like to not rely on third-party fields if we don't have to and roll our own but ⏲️

Reply:
Fair enough, I'll spin up a ticket for it for later